src/pages/wc.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect } from 'react'
+import type { NextPage } from 'next'
+import { useRouter } from 'next/router'
+import { parse } from 'querystring'
+import useLastSafe from '@/hooks/useLastSafe'
+import { AppRoutes } from '@/config/routes'
+
+const WcPage: NextPage = () => {
+  const router = useRouter()
+  const lastSafe = useLastSafe()
+
+  useEffect(() => {
+    if (!router.isReady || router.pathname !== AppRoutes.wc) {
```

Comment:
I think since this is always on the `/wc` page we don't need to check if the pathname matches that.

Reply:
I copied that from the index.tsx redirect. It's probably there to avoid the effect kicking in again after the redirect but before the page is unmounted.